.github/workflows/release.yaml

```diff
@@ -8,21 +8,31 @@ on:
       - .github/renovate.json5
       - pyproject.toml
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
+  get-version:
+    uses: ./.github/workflows/get-version.yaml
+
   ci:
     name: Build & Test
     uses: ./.github/workflows/build-and-test.yaml
     secrets: inherit
+    with:
+      snap_risk_level: edge
```

Comment:
making the `snap_risk_level` configurable as an input would be much better for the release workflow.
(we can do that in a separate PR.)